pynsee/sirene/sirenedataframe.py

```diff
@@ -56,6 +55,23 @@ def _warning_OSM():
 NOMINATIM_RESULTS = ["latitude", "longitude", "category", "type", "importance"]
 
 
+def clean_cities(s: pd.Series) -> pd.Series:
+    "Do some precleaning for cities names"
+    s = (
+        s.str.upper()
+        .apply(unidecode)
+        # Neuville-Housset (La) -> Neuville-Housset:
+        .str.replace(r" \(.*\)$", "", regex=True)
+        .str.replace(r"(^|\s)(ST)\s", " SAINT ", regex=True)
+        .str.replace(r"(^|\s)(STE)\s", " SAINTE ", regex=True)
+        # PARIS 4, PARIS 1er, LE TAMPON 14EME KM, etc. :
+        .str.replace(r"[0-9]+(ER?|EME)?( KM)?(\s|$)", "", regex=True)
```

Comment:
what's KM?

Reply:
"LE TAMPON 14EME KM" is an area in La Réunion. But that needs to be altered (we're not trying to cleanup the cities for itself, and that info might be useful for Nominatim)